pkg/account/server.go

```diff
@@ -50,27 +50,27 @@ type Component interface {
 
 type server struct {
 	c             Component
-	config        oauth.Config
+	config        Config
 	store         account_store.TransactionalInterface
 	session       sess.Session
 	generateCSP   func(config *oauth.Config, nonce string) string
 	schemaDecoder *schema.Decoder
 }
 
 // NewServer returns a new account app on top of the given store.
-func NewServer(c *component.Component, store account_store.TransactionalInterface, config oauth.Config, cspFunc func(config *oauth.Config, nonce string) string) (Server, error) {
+func NewServer(c *component.Component, store account_store.TransactionalInterface, config Config, cspFunc func(config *oauth.Config, nonce string) string) (Server, error) {
```

Comment:
I believe it generates a lint error with the new ci, since the line is too big.
If it is of interest, the link to Hylke's comment on how he breaks the methods: https://github.com/TheThingsNetwork/lorawan-stack/pull/5487#discussion_r880555535